src/compass/utils/geo_metadata.py

```diff
@@ -196,7 +206,8 @@ def from_file(cls, file_path: str, fmt: str):
                    meta_dict['range_window_type'],
                    meta_dict['range_window_coefficient'], cfg, geogrid,
                    meta_dict['nodata'], meta_dict['input_data_ipf_version'],
-                   meta_dict['isce3_version'])
+                   meta_dict['isce3_version'],
+                   meta_dict['eap_correction_by_opera'])
```

Comment:
I would simply write `antenna_patter_correction`. We are re-crafting the output product allocation as a HDF5 file, in the attribute description we will make sure to indicate that this is a Boolean flag indicating if the correction has been applied or not. Since you are including this in the metadata, no need to submit an issue (as previously indicated in my comment) but do remove the comment ;)

Reply:
I was also thought about `antenna_patter_correction`, but felt that it would be necessary to indicate if the correction is applied by ESA (i.e. IPF >= 2.43) or OPERA, so that we can track back if there are any issues regarding the EAP correction.

What do you think about using the field name `antenna_patter_correction`, and putting the string `ESA` or `OPERA`?

Reply:
Replacing the field name to Liang's suggestion.